### PR TITLE
Renamed Correlation and updated documentation

### PR DESCRIPTION
**Pull request by TimKDJ**
Correlation is now called correlation; I removed the pairs/matrix stuff.

I also updated the documentation to reflect the changes made in the jaspResults rewrite.
The Dutch documentation still needs to be changed, but I'll do that once I know what names the translators give the various interface options.

### CODE REVIEW

File: Resources/Help/analyses/correlationbayesian.md [hunk 1]
```diff
@@ -44,6 +46,23 @@ The Bayesian Correlation Matrix allows estimation of the population correlation,
...
+### Plot Individual Pairs
+- Correlation coefficient to plot:
+  - Pearson's rho: Pearson's product moment correlation coefficient. 
+  - Kendall's tau-b: Kendall's tau-b rank-order correlation coefficient to quantify the monotonic association between two variables.
+  
+- Scatterplot: Displays scatterplots for each specified pair of variables.
+  
+- Prior and posterior: Displays the prior and posterior distribution of the correlation under the alternative hypothesis for each specified pair of variables.
+  - Estimation info: Adds the median and the 95% credible interval of the posterior distribution of the effect size.
+  - Testing info: Adds the Bayes factor computed with the user-defined prior; adds a probability wheel depicting the odds of the data under the null vs. alternative hypothesis; adds indicators representing the prior and posterior at the test point (e.g., rho=0), the ratio between these two values equals the Bayes factor.
```
Comment (AlexanderLyNL) [inline Resources/Help/analyses/correlationbayesian.md]: Testing info: Adds the Bayes factor computed with the user-defined prior; adds a probability wheel depicting the odds of the data under the null vs. alternative hypothesis (under the assumption that the null and alternative hypotheses were equal probable a priori, i.e., P(H0)=P(H1)=1/2); adds indicators representing the prior and posterior at the test point (e.g., rho=0), the ratio between these two values equals the Bayes factor.

File: Resources/Help/analyses/correlationbayesian.md [hunk 2]
```diff
@@ -44,6 +46,23 @@ The Bayesian Correlation Matrix allows estimation of the population correlation,
...
+  
+- Bayes factor robustness check: Displays the Bayes factor as a function of the width of the stretched beta prior on the correlation for each specified pair of variables. The width of the beta prior is varied between 0 and 2.
```
Comment (AlexanderLyNL) [inline Resources/Help/analyses/correlationbayesian.md]: width kappa

File: Resources/Help/analyses/correlationbayesian.md [hunk 3]
```diff
@@ -67,18 +87,36 @@ The Bayesian Correlation Matrix allows estimation of the population correlation,
...
     - Posteriors under H1: Displays density plot(s) of the posterior of the correlation(s) under the alternative hypothesis in the lower off-diagonal entries of the matrix.
 
+#### Bayesian Correlation Pairwise Plots
+- Scatterplot:
+  - Displays the scatterplot of the correlation pairs.
+
+- Prior and Posterior:
+  - Displays the prior (dashed line) and posterior (solid line) distribution of the correlation under the alternative hypothesis; The horizontal solid line represents the width of the 95% credible interval of the posterior distribution.
+    - Estimation info: Displays the median and the 95% credible interval of the posterior distribution of the effect size.
+    - Testing info: Displays the Bayes factor computed with the user-defined prior; displays a probability wheel depicting the odds of the data under the null vs. alternative hypothesis; displays two gray circles representing the prior and posterior at correlation = 0.
```
Comment (AlexanderLyNL) [inline Resources/Help/analyses/correlationbayesian.md]: Should be the same as above

File: Resources/Help/analyses/correlationbayesian.md [hunk 4]
```diff
@@ -67,18 +87,36 @@ The Bayesian Correlation Matrix allows estimation of the population correlation,
...
+
 ### References
 ---
 - Jeffreys, H. (1961). *Theory of probability (3rd ed.)*. Oxford, UK: Oxford University Press.
 - Ly, A., Verhagen, A. J., & Wagenmakers, E.-J. (2016). Harold Jeffreys's default Bayes factor hypothesis tests: Explanation, extension, and application in psychology. *Journal of Mathematical Psychology, 72*, 19-31.
 - Ly, A., Marsman, M., & Wagenmakers, E.-J. (2018).  Analytic Posteriors for Pearson’s Correlation Coefficient. *Statistica Neerlandica, 72*(1), 4-13
 - Rouder, J. N., & Morey R. D. (2012). Default Bayes factors for model selection in regression. *Multivariate Behavioral Research, 47*, 877-903.
 - van Doorn, J.B., Ly, A., Marsman, M. & Wagenmakers, E.-J. (2018). Bayesian Inference for Kendall’s Rank Correlation Coefficient. *The American Statistician*,  72, 303-308.
+- Wagenmakers, E.-J., Lodewyckx, T., Kuriyal, H., and Grasman, R. (2010). Bayesian hypothesis testing for psychologists: A tutorial on the Savage-Dickey method. Cognitive Psychology, 60, 158-189.
```
Comment (AlexanderLyNL) [inline Resources/Help/analyses/correlationbayesian.md]: *Cognitive Psychology, 60*

File: Resources/Help/analyses/correlationbayesian_nl.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
-Bayesiaanse Correlatie Matrix
+Bayesiaanse Correlatie
 ===
 
-De Bayesiaanse Correlatie Matrix maakt het mogelijk de populatiecorrelatie te schatten en de nulhypothese dat de populatiecorrelatie tussen paren van variabelen gelijk is aan 0, te toetsen. Alle mogelijke paren van de gespecificeerde variabelen worden geanalyseerd.
+De Bayesiaanse Correlatie analyse maakt het mogelijk de populatiecorrelatie te schatten en de nulhypothese dat de populatiecorrelatie tussen paren van variabelen gelijk is aan 0, te toetsen. Alle mogelijke paren van de gespecificeerde variabelen worden geanalyseerd.
```
Comment (AlexanderLyNL) [inline Resources/Help/analyses/correlationbayesian_nl.md]: De Bayesiaanse Correlatie analyse maakt het mogelijk om de populatiecorrelatie te schatten en om hypothesen te vergelijken. De drie hypothesevergelijkingen zijn: (i) tussen de  nulhypothese (H0) dat de populatiecorrelatie tussen paren van variabelen gelijk is aan 0 en de alternatieve hypothese (H1) dat de populatiecorrelatie een (onbekende) waarde aanneemt tussen -1 en 1, (ii) tussen H0 en de alternatieve hypothese (H+) dat de populatiecorrelatie positief is, en (iii) tussen H0 en de alternatieve hypothese (H-) dat de populatie correlatie negatief is. Alle mogelijke paren van de gespecificeerde variabelen worden geanalyseerd.

**Review by AlexanderLyNL — CHANGES_REQUESTED** [review]
Looks great. I do feel that a consistency check is needed (to see if things are copy-pasted correctly), but other than that it looks great.

File: Resources/Help/analyses/correlationbayesian_nl.md (hunk 5 above)
Comment (TimKDJ) [reply Resources/Help/analyses/correlationbayesian_nl.md]: Heb je hier ook een mooi Engels zinnetje voor? ;)
Anders wijken de twee talen van elkaar af

File: Resources/Help/analyses/correlationbayesian.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
-Bayesian Correlation Matrix
+Bayesian Correlation
 ===
 
-The Bayesian Correlation Matrix allows estimation of the population correlation, as well as testing the null hypothesis that the population correlation between pairs of variables equals 0. All possible pairs of the specified variables are analyzed.
+The Bayesian Correlation analysis allows estimation of the population correlation, as well as testing the null hypothesis that the population correlation between pairs of variables equals 0. All possible pairs of the specified variables are analyzed.
```
Comment (AlexanderLyNL) [inline Resources/Help/analyses/correlationbayesian.md]: De Bayesiaanse Correlatie analysis allows for the estimation of the population correlation, as well as the comparison of hypotheses. The three comparisons are (i) between the null hypothesis (H0) that the correlation between pairs of variables equal 0, and the alternative hypothesis (H1) that the population correlation takes its value between -1 and 1; (ii) between H0 and the alternative hypothesis (H+) that the population correlation is positive; and (iii) between H0 and the alternative hypothesis (H-) that the population correlation is negative. All possible pairs of the specified variables are analyzed.

Comment (TimKDJ) [reply Resources/Help/analyses/correlationbayesian.md]: Thanks -- copy & pasted (minus the first three words :))

**Review by AlexanderLyNL — APPROVED** [review]

**Pull request merged**